src/web/multi-target.md

```diff
@@ -0,0 +1,35 @@
+# Multi-Target (targeting both native and WASM)
+
+It is possible to compile a project for both native systems (i.e. Windows,
+MacOS and/or Linux) and WebAssembly without the use of custom feature flags.
+The following snippet from `Cargo.toml` works in Rust nightly.
+
+```toml
+# This part only gets compiled for native.
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+bevy = {version = "0.4", default-features = false, features = ["bevy_dynamic_plugin", "bevy_wgpu", "bevy_winit", "png", "render", "x11", "wayland"]}
```

Comment:
I'd suggest advising users to set these required features in the features section rather than in the import itself.  This seems to be more compatible with the directions below. I was unable to get this method to work.

Reply:
This probably didn't work because I forgot to document a step here. Fixed in c163afe. Additionally I future-proofed this part a bit more, as the required feature will end up in stable Rust in less than 2 weeks.

As for the features section, I assume you mean something like this instead?

```
[features]
default = [
  "bevy/bevy_gltf",
  "bevy/bevy_winit",
  "bevy/render",
  "bevy/png",
]

native = [
  "bevy/bevy_wgpu",
]

web = [
  "bevy_webgl2"
]
```

This is specifically what this multi-target scenario is trying to avoid.